GeoQuiz/app/src/main/java/com/example/geoquiz/QuizActivity.kt

```diff
@@ -25,18 +33,12 @@ class QuizActivity : AppCompatActivity() {
     private lateinit var scoreButton: Button
 
 
+    private val quizViewModel: QuizViewModel by lazy{
+        ViewModelProviders.of(this).get(QuizViewModel::class.java)
+    }
 
-    private var mCurrentIndex: Int=0
-
-    private var mQuestionBank= listOf<Question>(
-        Question(R.string.question_australia,true),
-        Question(R.string.question_oceans, true),
-        Question(R.string.question_mideast, false),
-        Question(R.string.question_africa, false),
-
-    )
 
-    private var clicked:BooleanArray= BooleanArray(mQuestionBank.size)
+    private var clicked:BooleanArray= BooleanArray(4)
```

Comment:
The legend says even the best developers still struggle finding good and intuitive variable names. Still, this race to perfection can be started from early stages. I think this one can be improved. I can see this relates to something being clicked, but what is that? Is it a button? Is it an image? Is this the representation of a question being clicked/shown? Please rename this to better reflect it's purpose